app/http.php

```diff
@@ -411,11 +411,12 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
     ->param('cpus', 1, new Integer(), 'Container CPU.', true)
     ->param('memory', 512, new Integer(), 'Comtainer RAM memory.', true)
     ->param('version', 'v4', new WhiteList(['v2', 'v4']), 'Runtime Open Runtime version.', true)
+    ->param('autoRestart', false, new Boolean(), 'Runtime Open Runtime version.', true)
```

Comment:
Why dont we use `restartPolicy` like we did for orchestration ? 
also description needs to be updated